packages/ui/src/council/types/PastElection.ts

```diff
@@ -28,7 +29,7 @@ export interface PastElectionWithDetails extends PastElection {
 export const asPastElection = (fields: PastElectionRoundFieldsFragment): PastElection => ({
   id: fields.id,
   cycleId: fields.cycleId,
-  finishedAt: fields.updatedAt,
+  finishedAtBlock: asBlock((fields.referendumResult as any[])[0].referendumFinishedEvent),
```

Comment:
My take on this:
```suggestion
  finishedAtBlock: fields.referendumResult?.[0] && asBlock(fields.referendumResult[0].referendumFinishedEvent),
```
The type of `finishedAtBlock` is `Block | undefined` then, and `undefined` could indicate an unknown block, and display as an empty component or whatever we want to display for it.

Reply:
Also today I've learned that the expression `null?.[0]` is certifiably cursed